core/modules/system/system.module

```diff
@@ -1114,6 +1113,16 @@ function system_menu() {
     'theme callback' => 'ajax_base_page_theme',
   );
 
+  // AJAX endpoint for retrieving a token.
+  $items['token-browser/token/%'] = array(
+    'page callback' => 'system_token_browser_endpoint',
+    'page arguments' => array(2),
+    'delivery callback' => 'system_token_browser_endpoint_output',
+    'access callback' => 'system_token_browser_endpoint_access',
```

Comment:
I did a search on "endpoint" in core and it's not a term we've used anywhere else. Let's shorten these function names up:
```suggestion
    'page callback' => 'system_token_browser',
    'page arguments' => array(2),
    'delivery callback' => 'system_token_browser_output',
    'access callback' => 'system_token_browser_access',
```

Reply:
Done